elixir/apps/domain/lib/domain/types/ip_port.ex

```diff
@@ -44,7 +44,7 @@ defmodule Domain.Types.IPPort do
   def cast_address(address) do
     address
     |> String.to_charlist()
-    |> :inet.parse_address()
+    |> :inet.parse_strict_address()
```

Comment:
We need a test for parsing since we started hitting bugs with it

Reply:
Added a couple of tests for the `IPPort` type.